src/utils/hashing_encoding.ts

```diff
@@ -0,0 +1,19 @@
+export function base64url(binary: Uint8Array): string {
+  const binaryString = Array.from(binary).map((b) => String.fromCharCode(b))
+    .join("");
+  const output = btoa(binaryString);
+  const urlSafeOutput = output
+    .replaceAll("=", "")
+    .replaceAll("+", "-")
+    .replaceAll("/", "_");
+  return urlSafeOutput;
+}
```

Comment:
Can we use https://deno.land/std@0.212.0/encoding/base64url.ts?s=encodeBase64Url?

Reply:
I guess so, although this is out of scope of this PR, just moved it in here from `access_token.ts`.

Reply:
Sure, that makes sense. Let's do that later